client/client.go

```diff
@@ -130,8 +131,21 @@ type doer interface {
 type Config struct {
 	// BaseURL contains the base URL where the Pebble daemon is expected to be.
 	// It can be empty for a default behavior of talking over a unix socket.
+	// If the protocol prefix is https://, TLS will be used.
 	BaseURL string
 
+	// VerifyTLSConnection provides the client with an opportunity to inspect
+	// and optionally reject the incoming server certificate.
+	VerifyTLSConnection func(tls.ConnectionState) error
+
+	// Optional HTTP basic authentication details. If supplied this will
+	// add an HTTP basic authentication header entry.
+	// RFC 7617 (HTTP Authentication: Basic and Digest) support a user without
+	// a password, but we will error if an empty password is provided for
+	// security reasons.
+	BasicAuthUser string
+	BasicAuthPass string
```

Comment:
@benhoyt CTO was not yet happy with the names. If you have a shorter and nicer proposal, please let me know.

Reply:
Matching the shortened version in the identities package, I call them BasicUser and BasicPass now.

Reply:
Resolving.